test/optimise.jl

```diff
@@ -1,42 +1,62 @@
-using IntervalArithmetic, IntervalOptimisation
+﻿using IntervalArithmetic, IntervalOptimisation
 using Test
 
 @testset "IntervalOptimisation tests" begin
 
-    @testset "Minimise in 1D" begin
+    @testset "Minimise in 1D using default data structure i.e HeapedVector" begin
         global_min, minimisers = minimise(x->x, -10..10)
         @test global_min ⊆ -10 .. -9.999
         @test length(minimisers) == 1
         @test minimisers[1] ⊆ -10 .. -9.999
 
-        global_min, minimisers = minimise(x->x^2, -10..11, 1e-10)
+        global_min, minimisers = minimise(x->x^2, -10..11, tol = 1e-10)
         @test global_min ⊆ 0..1e-20
         @test length(minimisers) == 1
         @test minimisers[1] ⊆ -0.1..0.1
 
         global_min, minimisers = minimise(x->(x^2-2)^2, -10..11)
         @test global_min ⊆ 0..1e-7
         @test length(minimisers) == 2
-        @test sqrt(2) ∈ minimisers[1]
+        @test sqrt(2) ∈ minimisers[2]
     end
 
+    for Structure in (SortedVector, HeapedVector)
 
-    @testset "Discontinuous function in 1D" begin
+        @testset "Minimise in 1D using SoretedVector" begin
```

Comment:
SortedVector

Reply:
This is wrong now -- you should substitute in the value of `structure` (small `s`).

Reply:
Sorry I dont get it